docs/windows.rst

```diff
@@ -4,26 +4,43 @@ Windows Guide
 
 Installation
 ================
-To install xonsh on Windows, first install `Python v3.4+`_ from
-http://python.org .
 
-Next, install the other dependencies via ``pip``:
+The easy way
+----------------
+
+The easiest way to install xonsh on windows is through the `Anaconda Python Distribution`_ and the conda package manager. 
+
+.. note:: Be sure to install the version with Python3.4 or 3.5. Xonsh does is not supported on legacy Python (2.7). 

```

Comment:
Maybe you can say `Python v3.4 and later`, so that this doesn't go out of date with v3.6 :)